src/olympia/reviewers/tests/test_utils.py

```diff
@@ -2139,6 +2166,30 @@ def test_reject_multiple_versions(self):
         assert not flags.auto_approval_disabled_until_next_approval_unlisted
         assert flags.auto_approval_disabled_until_next_approval
 
+    def test_reject_multiple_versions(self):
+        self._test_reject_multiple_versions({})
+        message = mail.outbox[0]
+        assert message.subject == (
+            'Mozilla Add-ons: Delicious Bookmarks has been disabled on '
+            'addons.mozilla.org'
+        )
+        assert 'your add-on Delicious Bookmarks has been disabled' in message.body
+
+    def test_reject_multiple_versions_resolving_abuse_report(self):
```

Comment:
this split of the test was to test the emails went out correctly when a job was being resolved.  I just did this action - it should work the same for each - but I can repeat for the others if you think it's necessary?